src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,42 @@
+.nav {
+  &__link,
+  &__list {
+    display: flex;
+  }
+
+  &__list {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    position: relative;
+    align-items: center;
+    height: 60px;
+    font-size: 12px;
+    text-decoration: none;
+    text-transform: uppercase;
+    white-space: nowrap;
+    color: #060b35;
+  }
+
+  &__link:hover,
+  &__link.is-active {
+    color: #00acdc;
+  }
+
+  &__link.is-active::after {
+    content: "";
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 4px;
+    background: #00acdc;
+    border-radius: 2px;
+  }
```

Comment:
Remove these selectors inside `&__link`, so instead of writing `&__link:hover` you go with `&:hover`